client/test/suite/index.ts

```diff
@@ -0,0 +1,37 @@
+import * as path from 'path';
+import Mocha from 'mocha';
+import { glob } from 'glob';
+
+export function run(): Promise<void> {
```

Comment:
It would be nice to have these run in CircleCI as well.

Reply:
Let me update the CI configuration.